frontend/src/App.js

```diff
@@ -42,6 +47,11 @@ export default function App() {
               selectionParameters={selectionParameters}
             />
           </Route>
+          <Route exact path="/scoreOverview">
+            <Layout>
```

Comment:
Imho it's a bit odd that only some pages use `Layout`. Maybe incorporate it into the corresponding pages to shorten this file?